upstream-community-operators/federation/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: federation.v0.0.6
+  name: federation.v0.0.7
   namespace: placeholder
   annotations:
     capabilities: Basic Install
     categories: "OpenShift Optional, Integration & Delivery"
     description: Gain Hybrid Cloud capabilities between your clusters with Kubernetes Federation.
     certified: "false"
-    containerImage: "quay.io/openshift/origin-federation-controller:v4.0.0"
+    containerImage: "quay.io/kubernetes-multicluster/federation-v2:v0.0.7"
     createdAt: "2019-01-01T00:00:00Z"
 spec:
```

Comment:
Please add a `spec.links` section with links to external documentation and other links relevant to this operator